### DISCUSSION TITLE
Input the volume increment

### DISCUSSION THREAD

**Discussion by HEPBO3AH** [MonitorControl/MonitorControl#1560] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Is your feature request related to a problem? Please describe

On my LG monitor the default OSD scale for volume is 6 which is too high.

Selecting fine control checkbox makes it 1, which is too low.

### Describe the solution you'd like

It would be nice to have it as a number/multiplier field/slider to have fine adjustment and make it *just right*!

### Describe alternatives you've considered

There isn't really any alternative that I'm aware of, please let me know if there is a config file I can use to achieve this

**Comment by waydabber** [comment]
Hi, you can change the DDC volume control range under advanced settings for the display. Try to change the default 0-100 to something else so the range is clipped.
